visualization-requirements.typ

```diff
@@ -30,7 +30,7 @@ The load balancer will be showcased at the innovation expo in April. To help com
 
 == Default
 
-A figma mockup and description of the default visualization:
+The purpose of the default visualization is to show functionality and overall of the balancer to an audience. As a result, it's expected that only one balancer will be on-screen at any given time for this visualization since its function is to communicate the concept of a load balancer and how it applies to OpenTogetherTube.
```

Comment:
```suggestion
The purpose of the default visualization is to show functionality and overall state of the balancer to an audience. As a result, it's expected that only one balancer will be on-screen at any given time for this visualization since its function is to communicate the concept of a load balancer and how it applies to OpenTogetherTube.
```